.github/workflows/ci-pulsarbot.yaml

```diff
@@ -41,6 +41,28 @@ jobs:
 
       - name: Execute pulsarbot command
         id:   pulsarbot
+        if: github.event_name == 'issue_comment' && startsWith(github.event.comment.body, '/pulsarbot')
         env:
           GITHUB_TOKEN: ${{ secrets.PULSARBOT_TOKEN }}
         uses: apache/pulsar-test-infra/pulsarbot@master
+
+      - name: Cherry-pick prompt
+        if: github.event_name == 'pull_request' && github.event.pull_request.merged == true
+        uses: apache/pulsar-test-infra/cherry-pick@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.PULSARBOT_TOKEN }}
+        with:
+          type: prompt-comment
+          github_user: codelipenghui
```

Comment:
Just use a github user who has permission to create a branch on the apache/pulsar repo. We can consider creating a bot user to do this.

Reply:
Ok, we can update it later

Reply:
@zymap Can you leave a comment in the file?

Reply:
ok.